README.md

```diff
@@ -51,8 +51,8 @@ Run in your console: `npm install medium-editor`
  For a custom one:
 
  ```html
- <script src="//cdn.jsdelivr.net/medium-editor/4.11.1/js/medium-editor.min.js"></script>
- <link rel="stylesheet" href="//cdn.jsdelivr.net/medium-editor/4.11.1/css/medium-editor.min.css" type="text/css" media="screen" charset="utf-8">
+ <script src="//cdn.jsdelivr.net/npm/medium-editor@5.23.2/dist/js/medium-editor.min.js"></script>
```

Comment:
@LukasDrgon  why not we change the `src` to use this `src="//cdn.jsdelivr.net/npm/medium-editor@latest/dist/js/medium-editor.min.js"` ?

Reply:
@rishijain if you check the full readme, you have two versions of the links there. One with `latest` and one with a specific version.

Reply:
ah my bad. completely missed it. Thanks for the change.

Reply:
@MartinKolarik @LukasDrgon maybe you can change the path to the latest version to use the new system

Reply:
@j0k3r #1367